analyzers/tests/SonarAnalyzer.Test/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.FlowCaptureOperation.cs

```diff
@@ -0,0 +1,372 @@
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarAnalyzer.Test.LiveVariableAnalysis;
+
+public partial class RoslynLiveVariableAnalysisTest
+{
+    [TestMethod]
+    public void FlowCaptrure_NullCoalescingAssignment()
+    {
+        /*      Block 1
+        *      #0=param
+        *          |
+        *       Block 2
+        *        #1=#0
+        *      #1 is null --+
+        *          |        |
+        *       Block 3     |
+        *       #0=Hello    |
+        *          |        |
+        *        Exit <-----+
+        */
+        const string code = """param ??= "Hello";""";
+        var context = CreateContextCS(code, additionalParameters: "string param");
+        context.ValidateEntry(LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[1], LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[2], LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[3], LiveIn("param"));
+        context.ValidateExit();
+    }
+
+    [TestMethod]
+    public void FlowCaptrure_NullCoalescingOperator()
+    {
+        /*       Block 1
+         *      #0=param
+         *     #0 is null
+         *        /   \
+         *       F     T
+         *      /       \
+         *  Block 2    Block 3
+         *  #1=#0     #1="Hello"
+         *       \     /
+         *       Block 4
+         *      result=#1
+         *         |
+         *        Exit
+         */
+        const string code = """var result = param ?? "Hello";""";
+        var context = CreateContextCS(code, additionalParameters: "string param");
+        context.ValidateEntry(LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[1], LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[2], LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[3], LiveIn("param"), LiveOut("param"));
+        context.Validate(context.Cfg.Blocks[4], LiveIn("param"));
+        context.ValidateExit();
+    }
+
+    [TestMethod]
+    public void FlowCaptrure_ConditionalAccess()
+    {
+        /*       Block 1
+         *      #1=param
+         *     #1 is null
+         *        /   \
+         *       F     T
+         *      /       \
+         *  Block 2    Block 3
+         * #0=Length  #0=default
```

Comment:
This part is still missing 
```suggestion
         * #0=#1.Length  #0=default
```